src/validators/typings.py

```diff
@@ -36,12 +36,12 @@ class ExitSignatureShards:
 
 @dataclass
 class OraclesApproval:
-    validators_registry_root: Bytes32
     signatures: bytes
     ipfs_hash: str
 
 
 @dataclass
+# pylint: disable-next=too-many-instance-attributes
```

Comment:
(not now) Looks like linter wants to see `list[OracleData]` where OracleData is `dataclass(public_key, deposit_signature, public_key_shards, exit_signature_shards)`